openfisca_us/variables/gov/states/il/tax/income/personal_exemption/il_is_exemption_eligible.py

```diff
@@ -0,0 +1,21 @@
+from openfisca_us.model_api import *
+
+
+class il_is_exemption_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Whether this tax unit is eligible for any exemptions"
+    unit = USD
+    definition_period = YEAR
+    reference = ""
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.il.tax.income.personal_exemption.cap
+        filing_status = tax_unit("filing_status", period)
+        joint = where(
```

Comment:
Replace these with breakdown parameters for every filing status (duplicate data but more readable code and works better with PolicyEngine)

Reply:
[joint/nonjoint > filing statuses in cap.yaml](https://github.com/PolicyEngine/openfisca-us/pull/1056/commits/08a3e308b07a794b0bd60c68632790d9cf619c42)